src/Slices/ServiceInventory/UI/ServiceInventory.test.tsx

```diff
@@ -144,70 +88,63 @@ function setup(service = Service.a, pageSize = "") {
     scheduler,
   };
 }
+const server = setupServer();
+
+beforeAll(() => server.listen());
```

Comment:
Doesn't this normally fit in a describe function and then have each case with an it?
 Seeing it like this, I fear it might leak in other tests or have unexpected behavior🤔

Reply:
I will move that